lib/api/entities.rb

```diff
@@ -114,6 +114,19 @@ class Namespaces < Grape::Entity
         type: "Boolean",
         desc: "Whether this is the global namespace or not"
       }
+      expose :permissions, documentation: {
+        desc: "Different permissions for the current user"
+      } do |namespace, options|
+        user = options[:current_user]
+        # TODO: taken from NamespacesHelper. Avoid duplication! (e.g. owner?)
```

Comment:
I thought about this but I couldn't find a way to inject helpers to be used inside an entity. I guess we'll need to inject stuff on model and reuse it here. I dunno know. We definitely need to find a better solution for this.